docs/control_panel_integration/README.md

```diff
@@ -77,9 +77,15 @@ That means that you can add `Conficts: public_cp_vendors_api < VERSION` to the s
 
 ### Changelog
 ::: tip Changelog
+
+Version 1.2
+1. `Provides public_cp_vendors_api = 1.2` added to rpm spec of alt-python27-cllib package (see [versioning](/control_panel_integration/#versioning)).
+2. Possibilty to run [`domains` integration script](https://docs.cloudlinux.com/control_panel_integration/#domains) with optional flag `--with-php` added.
+3. New X-Ray feature added to `supported_cl_features` of [`panel_info` script](/control_panel_integration/#panel-info). Now the X-Ray UI tab is hidden until the panel makes the support clear by setting the `xray` feature to `true`.
```

Comment:
By default

Reply:
> support clear by setting the `xray` feature to `true`
but we still support old integrations which do not provide that key, right? Vendors don't have to update their scripts of they don't need x-ray?

Reply:
Added.
No updates needed unless a vendor decides to integrate X-Ray.